core/src/main/java/com/scalar/db/common/error/CoreError.java

```diff
@@ -1079,6 +1087,12 @@ public enum CoreError implements ScalarDbError {
       "Something went wrong while scanning. Are you sure you are running in the correct transaction mode? Details: %s",
       "",
       ""),
+  DATA_LOADER_CONVERT_TO_STRING_FAILED(
```

Comment:
It seems that this message is no longer needed.

Reply:
I have removed it in [2828244](https://github.com/scalar-labs/scalardb/pull/2450/commits/28282448485f68192a1a41157334d01e593cebc9).
Thank you.